### DISCUSSION TITLE
Excluding autoexpansion in certain environments

### DISCUSSION THREAD

**Discussion by phantamoe** [artisticat1/obsidian-latex-suite#189] [Q&A]
When typing something in \text{} in math mode. I noticed that the autoexpansion of snippets is disabled. Is there any way to do disable this as well in some other commands such as \operatorname{}? I did not find a setting for it in obsidian.

**Comment by superle3** [comment]
There is no such setting yet, there is an pr open, but still unsure if it should be configurable, like who would want snippets turned on in operator name.
